meshroom/ui/qml/ImageGallery/ImageGallery.qml

```diff
@@ -34,12 +34,16 @@ Panel {
     title: "Image Gallery"
     implicitWidth: (root.defaultCellSize + 2) * 2
 
+    onReadOnlyChanged: {
```

Comment:
I don't think this is necessary since the `m.readOnly` property already depends on `root.readOnly` (the QML engine should update it dynamically).
Same for `onCurrentCameraInitChanged` below.

Reply:
This wasn't fully reliable before accessing `currentCameraInit.locked` (I'm not entirely sure why) but it is now. Thanks for your help!